includes/upload_page.form.inc

```diff
@@ -27,6 +27,7 @@ function islandora_paged_content_upload_page_form(array $form, array &$form_stat
   $upload_size = min((int) ini_get('post_max_size'), (int) ini_get('upload_max_filesize'));
   $extensions = array('tiff tif jp2 jpg jpeg');
   $languages = module_exists('islandora_ocr') ? islandora_ocr_get_enabled_tesseract_languages() : array();
+  $default_language = in_array('English', $languages) ? 'eng' : NULL;
```

Comment:
@qadan sorry 😢 . So this does not work.
islandora_ocr_get_enabled_tesseract_languages()  will get you something like
````
vagrant@islandora:~$ /usr/bin/tesseract --list-langs
List of available languages (4):
equ
eng
osd
fra
````
So no "English" in the array. Can you fix this?

Reply:
Whoops yeah, using the value instead of the key. Nice catch!